.travis.yml

```diff
@@ -3,7 +3,7 @@ language: php
 dist: trusty
 
 php:
-  - 7.1
+  - 5.6
```

Comment:
Isn't it possible to test with both versions? via the matrix? I don't know enough about it, but just seems like it would be worth considering.